app/views/profile/show.html.haml

```diff
@@ -0,0 +1,9 @@
+.inset
+  %h2= "Welcome, #{current_user.full_name} <#{current_user.email_address}>"
+  = form_tag(me_path(current_user), method: :post) do
+    %label.string.control-label First Name
+    %input.form-control{type: 'text', name: 'first_name', value: current_user.first_name}
```

Comment:
ok! yeah let's discuss. As a developer on this project, I'd find it easier to understand the default PATHs rather than `me`, but if there is a good user-facing reason to not use those I am all :ear: